route-rs-graphgen/src/main.rs

```diff
@@ -15,7 +15,7 @@ use std::collections::HashMap;
 mod codegen;
 mod pipeline_graph;
 
-enum ElementLink {
+enum SyncLink {
```

Comment:
This actually should be generic, I just don't have other cases yet.

Reply:
Okay, should the name be changed back? Or should it just be Link?

Reply:
It could just be `Link`, if that doesn't feel too terse.